pkg/cmd/roachtest/tests/disk_full.go

```diff
@@ -38,6 +39,17 @@ func registerDiskFull(r registry.Registry) {
 			c.Put(ctx, t.Cockroach(), "./cockroach", c.Range(1, c.Spec().NodeCount))
 			c.Start(ctx, t.L(), option.DefaultStartOpts(), install.MakeClusterSettings(), c.Range(1, nodes))
 
+			// Node 1 will soon be killed, when the ballast file fills up its disk. To
+			// ensure that the ranges containing system tables are available on other
+			// nodes, we wait here for at least two replicas of each range. Without
+			// this, it's possible that we end up deadlocked on a system query that
+			// requires a range on node 1, but node 1 will not restart until the query
+			// completes.
+			db := c.Conn(ctx, t.L(), 1)
+			err := WaitForReplication(ctx, t, db, 2)
```

Comment:
wait, why `2`? If you lose one of two replicas the range is still unavailable. Don't you need `3` here?

Reply:
#78617